saleor/payment/utils.py

```diff
@@ -67,6 +69,33 @@
 GENERIC_TRANSACTION_ERROR = "Transaction was unsuccessful"
 ALLOWED_GATEWAY_KINDS = {choices[0] for choices in TransactionKind.CHOICES}
 
+ALLOWED_MISSING_PSP_REFERENCE_MAP = {
```

Comment:
Double negative, probably changing to `PSP_REFERENCE_REQUIRED_MAP` is cognitively easier

Reply:
Good point, would be clearer, I'll change that.